control_panel_api/services.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+
+from . import aws
```

Comment:
Stupid question: Is this equivalent to `import aws`? Actually probably more like `from control_panel_api import aws`?

Reply:
Done like this so that if there is a library named `aws` it won't confilct